pkg/action/pull.go

```diff
@@ -138,6 +140,34 @@ func (p *Pull) Run(chartRef string) (string, error) {
 		return out.String(), err
 	}
 
+	// Print a pull summary for repo mode and non-OCI downloads.
+	// Some repos (e.g., Bitnami) resolve --repo pulls to oci:// refs, but that
+	// path does not go through the registry client's summary printer, so we emit
+	// a consistent summary here. We mirror the OCI output format:
+	//
+	//   Pulled: <chartUrl>:<version>
+	//   Digest: sha256:<digest>
+	//
+	// For HTTP/repo pulls, the digest is the SHA-256 of the downloaded .tgz archive.
+	// For direct OCI pulls, the registry client already prints "Pulled:" and
+	// "Digest:" (manifest digest), so we do not print here to avoid duplicates.
+	if p.RepoURL != "" || !registry.IsOCI(downloadSourceRef) {
+		base := trimAnySuffix(downloadSourceRef, ".tar.gz", ".tgz")
+		chart, ver := splitChartNameVersion(base)
+
+		tag := chart
+		if ver != "" {
+			tag += ":" + ver
+		} else if p.Version != "" {
```

Comment:
I am not sure we want this. Because it mean we can select a version but another one could be downloaded during resolving... Even less safe that url version extraction. I am ok to remove it or move the logic to somewhere else, but if we want to read the exact version we will have to use `chart/loader`.